Formula/zrok.rb

```diff
@@ -0,0 +1,31 @@
+require "language/node"
+
+class Zrok < Formula
+  desc "Geo-scale, next-generation sharing platform built on top of OpenZiti"
+  homepage "https://zrok.io"
+  url "https://github.com/openziti/zrok.git",
+    tag: "v0.3.5", revision: "917891e9b1fbe28e112ff7e399a92d60080c79e4"
```

Comment:
```suggestion
  url "https://github.com/openziti/zrok.git",
      tag:      "v0.3.5",
      revision: "917891e9b1fbe28e112ff7e399a92d60080c79e4"
```

Reply:
Just wanted to confirm the output of `brew audit` when moving revision to a new line:
```
zrok:
  * 7: col 5: Align the keys and values of a hash literal if they span more than one line.
Error: 1 problem in 1 formula detected
```

Reply:
yeah, as you can see, the change I suggested the two lines for `tag` and `revision` are aligned 😅

Reply:
Oh my, I can't read apparently. No I did not see that thanks for spelling it out for me! 😓